File: app/src/main/java/com/tripsta/tripstapoker/card/CardComponent.java
package com.tripsta.tripstapoker.card;

import javax.inject.Singleton;

import dagger.Component;

/**
 * Created by georgiossarris on 5/24/17.
 */

@Singleton
@Component(modules = {CardPresenterModule.class})
public interface CardComponent {
	void inject(CardActivity cardActivity);
}

File: app/src/main/java/com/sarris/poker/card/CardComponent.kt
package com.sarris.poker.card

import dagger.Component
import javax.inject.Singleton

/**
 * Created by georgiossarris on 5/24/17.
 */
@Singleton
@Component(modules = [CardPresenterModule::class])
interface CardComponent {
    fun inject(cardActivity: CardActivity?)
}
